hudi-common/src/test/java/org/apache/hudi/common/table/read/buffer/TestKeyBasedFileGroupRecordBuffer.java

```diff
@@ -168,7 +168,8 @@ void readWithEventTimeOrderingWithRecords() throws IOException {
     // update for 4 is ignored due to lower ordering value.
     // record5 is deleted.
     // delete for 6 is ignored due to lower ordering value.
-    assertEquals(Arrays.asList(testRecord1UpdateWithSameTime, testRecord2Update, testRecord3Update, testRecord4, testRecord6, testRecord7), actualRecords);
+    assertEquals(Arrays.asList(getSerializableIndexedRecord(testRecord1UpdateWithSameTime), getSerializableIndexedRecord(testRecord2Update),
```

Comment:
nitpick: you can use `Stream.of(...).map(getSerializableIndexedRecord)...` to make it a bit less verbose here and below